Makefile

```diff
@@ -11,17 +11,17 @@ ICON_THEMES := $(filter-out $(IGNORE), $(ICON_THEMES))
 all:
 
 install:
-	mkdir -p $(DESTDIR)$(PREFIX)/share/icons
-	cp -R $(ICON_THEMES) $(DESTDIR)$(PREFIX)/share/icons
+	mkdir -p "$(DESTDIR)$(PREFIX)/share/icons"
+	cp -R $(ICON_THEMES) "$(DESTDIR)$(PREFIX)/share/icons"
 
 # skip building icon caches when packaging
 	$(if $(DESTDIR),,$(MAKE) $(ICON_THEMES))
 
 $(ICON_THEMES):
-	-gtk-update-icon-cache -q $(DESTDIR)$(PREFIX)/share/icons/$@
+	-gtk-update-icon-cache -q "$(DESTDIR)$(PREFIX)/share/icons/$@"
 
 uninstall:
-	-rm -rf $(foreach icon_theme,$(ICON_THEMES),$(DESTDIR)$(PREFIX)/share/icons/$(icon_theme))
+	- rm -rf $(foreach icon_theme,$(ICON_THEMES),"$(DESTDIR)$(PREFIX)/share/icons/$(icon_theme)")
```

Comment:
Ignore the space that was added at the beginning of this line. I think I just unintentionally added it due to it looking a bit off to me.